Makefile

```diff
@@ -210,6 +210,9 @@ lint:
 	  tests/libvmi/... \
 	"
 
+lint-metrics:
```

Comment:
Don't we want to run this together with `lint`?

Reply:
@enp0s3 WDYT?